ghwatch.py

```diff
@@ -68,6 +71,10 @@
     ]},
 }
 
+# Priority list of notification reasons, from lowest to highest
+REASON_PRIO = ["subscribed", "state_change", "security_alert", "team_mention", "manual",
+               "invitation", "comment", "author", "mention", "review_requested", "assign"]
```

Comment:
nit, could be done later: as the label priorities are in descending order (highest to lowest), `REASON_PRIO` could be as well